README.md

```diff
@@ -1,24 +1,95 @@
-properly formatted JSON for Twitch overlays. This supports overlays for Game,
-Players, Queue, Ticker, Casters, and Lower Thirds. It pulls data from start.gg
-as well as Brawlhalla's Stats API.
+Gjallarhorn is a tool for Brawlhalla community tournament organizers to generate properly formatted JSON for Twitch overlays. Gjallarhorn can generate JSON for multiple use cases, including tickers, casters, lower thirds, queues, players, and games. It pulls data from start.gg as well as Brawlhalla's Stats API.
 
-## Usage
+## Getting Started
+
+You will need a start.gg API key to use Gjallarhorn. If you don't have one already, get one at https://developer.start.gg/docs/authentication.
 
-To start it up, you can run the following commands:
+To start Gjallarhorn, run the following commands in the project:
 
 ```
 npm run build
-node . -s [startgg-key-here]
+node . -s [your-startgg-key-here]
 ```
 
-Check out https://developer.start.gg/docs/authentication to get started with an
-API key if you don't have one already. You can fill out the cards with the
-proper details you want on the overlay, and hit "Push". You will find the output
-files served over http at http://localhost:3000/api/json or the raw json files
-in `packages/cli/output`
+Then, launch the dashboard at http://localhost:3000.
+
+## Usage
+
+You will find 6 different cards on the dashboard: Casters, Game, Ticker, Players, Queue, and Lower Thirds. You can fill out the cards with the details you want on the overlay and hit "Push". You will find the output files served over http at http://localhost:3000/api/json or the raw JSON files in `packages/cli/output`.
+
+![A Gjallarhorn interface filled in](images/gjallarhorn.png)
+
+You can reorder the cards by dragging the title bar. A dotted line will show where the card will be placed.
+
+![The Player card being reordered](images/gjallardrag.png)
+
+You can also recolor each card by right clicking on the title bar.
+
+![The Lower Thirds card with the color modal open](images/gjallarcolors.png)
+
+If a card is unnecessary or temporarily unneeded, you can minimize it by clicking the arrow at the right of each card next to the push button.
+
+These JSON files can be connected to broadcasting software (vMix or OBS with applicable plugins) to allow for seamless editing and usage of data. However, not all of this data will be useful or even applicable even if you are hosting a tournament.
+
+### Header
+
+In the top left corner, the current tournament is indicated next to the hamburger menu. The pencil allows you to change the active tournament. Enter the tournament’s unique slug (“brawlhalla-world-championship-expo-2023”) to switch to that tournament. Shorthand forms also exist for certain tournaments (“bcx”). Clicking on the game mode (In this example, Brawlhalla Championship 1v1) allows you to swap between different tournament game modes if they exist. The pools can also be swapped similarly.
```

Comment:
*pencil icon

Change "swap(ped)" to "change" or "switch". I think swap has a connotation that implies the thing you're taking out is goes in place of the thing you're putting in when really all we're concerned about is what's going in.